cfgov/jobmanager/tests/test_blocks.py

```diff
@@ -14,6 +14,7 @@ def setUp(self):
         self.root_page = Site.objects.get(is_default_site=True).root_page
         self.division = JobCategory.objects.create(job_category='Test')
         self.request = RequestFactory().get('/')
+        Site.find_for_request(self.request)
```

Comment:
I actually think we could remove this if you're not actually using the Site that's returned. That would reduce the query count described below.

Reply:
According to @chosak adding the `find_for_request` was his recommendation to reduce the query count.

Removing the `find_for_request` will increase the query count and produce the following:
AssertionError: 3 != 2 : 3 queries executed, 2 expected

Refer to https://docs.wagtail.io/en/stable/releases/2.9.html#sitemiddleware-and-request-site-deprecated
Please let me know how you want me to proceed.

Reply:
@cwdavies Yes, I understand that, but we're not using the `Site` that gets returned — it's a superfluous query here.

In the unit tests we could probably wrap the query count assertion in an `if wagtail.VERSION < (2, 9):` test.

Reply:
@willbarton Thanks for your feedback. I hope we can pair on handling the `requests.site` deprecation in another PR.
I value you opinion on when to use `if wagtail.VERSION < (2, 9):` in code as I do realize that `Site` is not being used and do not like have extra unnecessary code.